R/indent.R

```diff
@@ -78,18 +78,21 @@ indent_without_paren_for_while_fun <- function(pd, indent_by) {
 #' @describeIn update_indention Is used to indent if and if-else statements.
 #' @importFrom rlang seq2
 indent_without_paren_if_else <- function(pd, indent_by) {
+  expr_after_if_while <- next_non_comment(pd, which(pd$token == "')'")[1])
```

Comment:
So, why is the function called `indent_without_paren_if_else()`, and the code is looking for `"IF"` and `"WHILE"` tokens?

Reply:
Yes, that's inconsistent. I think it's no longer needed since the function `indent_without_paren_for_while_fun()` handles the `"WHILE"` case.

Reply:
Related: Since we now change 
```r
if (TRUE)
  return(call(a))
```
Into 
```r
if (TRUE) {
  return(call(a))
}
```
We should probably also change 
```r
while (FALSE)
  xyz(v)
```
Into
```r
while (FALSE) {
  xyz(v)
}
```
The same applies to `for`. Should we open an issue and postpone it?

Reply:
New issue and postponing sounds great. Do we fix the inconsistency in this PR?

Reply:
Oh, just saw the commit. Thanks!